src/main/java/frc/robot/commands/ShooterPrime.java

```diff
@@ -30,39 +30,31 @@ public ShooterPrime(ShooterSubsystem shooterSubsystem, LimelightSubsystem limeli
   // Called when the command is initially scheduled.
   @Override
   public void initialize() {
-    m_ShooterSubsystem.prime();
-    i = 0;
+    m_ShooterSubsystem.resetidelay();
   }
 
   // Called every time the scheduler runs while the command is scheduled.
   @Override
   public void execute() {
-    
+    m_ShooterSubsystem.windFlywheel(5.0);;
+  
   }
 
   // Called once the command ends or is interrupted.
   @Override
   public void end(boolean interrupted) {
     m_ShooterSubsystem.runCargo(false,false);
     m_ShooterSubsystem.windFlywheel(0);
-    m_CDSSubsystem.stopCDS();
-    SmartDashboard.putBoolean("wheelReady", false);
+    m_ShooterSubsystem.SetCargoBoolean(false);
+    
+    
+    //SmartDashboard.putBoolean("wheelReady", false);
   }
 
   // Returns true when the command should end.
   @Override
   public boolean isFinished() {
-    if(m_ShooterSubsystem.wheelReady()){
-      SmartDashboard.putBoolean("wheelReady", true);
-      //if(i > 0 || m_LimelightSubsystem.calculatePID() == 0.0){
-        m_ShooterSubsystem.runCargo(true,true);
-        m_CDSSubsystem.CDSBeltWheelControllerToggle(false);
-        i++;
-        if(i==100){ //Expected to add a 2000ms delay
-          return true;
-       // }
-      }
-    }
```

Comment:
Why did we get rid of this logic?

Reply:
It is moved from the command into the shootersubsystem (See windflywheel)